src/client/components/Bruincast/index.js

```diff
@@ -187,8 +186,16 @@ export const Bruincast = ({
       <Heading>{`Bruincast: ${course.title}`}</Heading>
       {warningElement}
       <Text>
-        Video recordings may take up to 24 hours before they are available.{' '}
-        <Link href="#">Help</Link>
+        Video recordings may take up to 24 hours before they are available. For
+        help regarding Bruincast content, please read our{' '}
+        <a
```

Comment:
@chern can’t you use the link component and pass in the rel attributes? See https://github.com/ucla/lti-library-resources/blob/master/src/client/components/Nav/index.js#L52